sceneview/src/main/java/io/github/sceneview/node/Node.kt

```diff
@@ -159,11 +154,13 @@ open class Node : NodeParent, TransformProvider, SceneLifecycleObserver {
     open var transform: Transform
         get() = translation(position) * rotation(quaternion) * scale(scale)
         set(value) {
-            position = Position(value.position)
-            quaternion = rotation(value).toQuaternion()
-            scale = Scale(value.scale)
+            position = Position(translation(value).position)
```

Comment:
Do we really need to extract a translation, rotation or scale matrix from the transform before converting it to a vector or quaternion?